config/routes.rb

```diff
@@ -1,5 +1,20 @@
 Rails.application.routes.draw do
-  root "homes#index"
+  devise_for :users
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root "monsters#index"
+  resources :monsters
+
+  get '/users/sign_out', to: "sessions#new"
```

Comment:
Was devise not redirecting successfully without this?  I don't think we need to specify the default controller it should point at